locale/en/manageiq.po

```diff
@@ -2794,10 +2796,20 @@ msgid "API Version"
 msgstr ""
 
 #:
-#: ../app/models/manageiq/providers/awx/automation_manager/amazon_credential.rb
+#: ../app/models/manageiq/providers/embedded_terraform/automation_manager/amazon_credential.rb
 msgid "AWS Access Key for this credential"
 msgstr ""
 
+#:
+#: ../app/models/manageiq/providers/embedded_terraform/automation_manager/amazon_credential.rb
+msgid "AWS Region"
+msgstr ""
+
+#:
+#: ../app/models/manageiq/providers/embedded_terraform/automation_manager/amazon_credential.rb
+msgid "AWS Region where the provider will operate. The Region must be set."
```

Comment:
simplified in https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/21